packages/fortress/static/main.css

```diff
@@ -0,0 +1,173 @@
+:root {
+  --key-color: #20123a;
+  --block-size: 16px;
+  --wild-gradient: repeating-linear-gradient(
+    -135deg,
+    #9059ff,
+    #d74cf0,
+    #ffd567,
+    #ffd567,
+    #d74cf0,
+    #9059ff
+  );
+  --highlight: #9059ff;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: metropolis;
+  margin: 0;
+  padding: 64px 0;
+  height: 100vh;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  flex-direction: column;
+  color: var(--key-color);
+  margin: 0;
+}
+
+.icon-wrap {
+  background-color: var(--key-color);
+  margin: 0 0 calc(2 * var(--block-size));
+}
+
+.icon {
+  width: calc(4 * var(--block-size));
+  height: calc(4 * var(--block-size));
+  background: url(/img/icon.svg) center center space;
+  background-size: calc(3 * var(--block-size));
+  transition: transform 150ms ease-out;
+}
+
+.icon-wrap:hover {
+  background-image: var(--wild-gradient);
+  background-size: 200%;
+  animation: spin 1s linear infinite;
+}
+
+h1 {
+  font-size: 54px;
+  margin: 0 0 20px;
+}
+
+h2 {
+  font-size: 32px;
+  margin: 0;
+  opacity: 0.7;
+}
+
+h3 {
+  font-size: 24px;
+  margin: 0 0 20px;
+}
+
+h4 {
+  margin: 0 0 24px;
+}
+
+.cards {
+  display: flex;
+  margin: 36px;
+}
+
+.card {
+  border: 3px solid var(--key-color);
+  padding: 24px;
+  border-radius: 4px;
+  width: 320px;
+  margin: 0 24px;
+}
+
+.popular {
+  box-shadow: 0 5px 18px rgba(0, 0, 0, 0.2);
+}
+
+.card.price-card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.card.price-card > * {
```

Comment:
This really doesn't matter for this package, and is irrelevant in the grand scheme of performance things, but star selectors are bad CSS selectors performance wise since selectors are processed by the browser from right-to-left. Plus since this is all direct children, if your HTML changes this might do funny things, but again, not a big deal for this package, more of a tip.